content/blog/2017-02-27-faas-serverless-enabler.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Challenges of Serverless in 2017"
+date = "2017-02-27"
+draft = "false"
++++
+
+Multi part Series on my thoughts on Serverless and its challenges for this year.
+
+Definition of Serverless has been a moving target. Since the launch of Lambda and the start of Serverless as a term we've seen a lot of different ways it has been described. AWS changed their messaging around Serverless last ReInvent going from Serverless == FaaS to Serverless encompasing many different and necessary services they provide. Lambda and FaaS for them is an enabler of Serverless, but not Serverless itself anymore. Check out [Tim Wagners talk on the State of Serverless](https://www.youtube.com/watch?v=AcGv3qUrRC4) for more info there
+
+The main principles of Serverless infrastructure still hold though:
+
+1. Less Control
+2. Less Responsibility
+3. Increased Automation
+4. Gradual Escalation of Control and Responsibility when necessary
```

Comment:
I changed the sentence before to refer to building serverless infrastructure. Imho costs are an important factor when evaluation and for the success of Serverless infrastructure, but they play a smaller role when building the infrastructure (unless any specific part is prohobitively expensive)